File: src/main/java/com/aaros/sankeweb/game/model/Food.java
package com.aaros.sankeweb.game.model;

public class Food {
  private int x;
  private int y;

  public Food() {
    this.x = Board.WIDTH / (Board.WIDTH / 100);
    this.y = Board.HEIGHT / (Board.HEIGHT / 100);
  }

  public int getX() {
    return x;
  }

  public void setX(int x) {
    this.x = x;
  }

  public int getY() {
    return y;
  }

  public void setY(int y) {
    this.y = y;
  }
}

File: frontend/src/game/Food.ts
import Game from "./Game.js";

export default class Food {
    x: number;
    y: number;
    game: Game;

    constructor(game: Game, x: number, y: number) {
        this.x = x;
        this.y = y;
        this.game = game;
    }
}
